spec/models/requests/requestable_decorator_spec.rb

```diff
@@ -1440,15 +1448,22 @@
       end
     end
 
+    context "at recap - firestone$pf" do
+      let(:stubbed_questions) { default_stubbed_questions.merge(recap?: true, recap_pf?: true) }
```

Comment:
This is an odd test, since for recap_pf? to be true recap? has to be true `recap? || recap_pf? || annex? || item_at_clancy? || held_at_marquand_library?` would evaluate to true as soon as recap? is true